src/window.ui

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk+" version="3.24"/>
+    <template class="WorldpossibleUpdutilWindow" parent="GtkApplicationWindow">
+      <property name="default-width">600</property>
+      <property name="default-height">300</property>
+      <child type="titlebar">
+        <object class="GtkHeaderBar" id="header_bar">
+          <property name="visible">True</property>
+          <property name="show-close-button">True</property>
+          <property name="title">World Possible Updutil</property>
+        </object>
+      </child>
+      <child>
+        <object class ="GtkBox">
+          <property name="visible">True</property>
+          <property name="orientation">vertical</property>
+          <property name="spacing">5</property>
+          <property name="margin">10</property>
+          <property name="valign">center</property>
+          <property name="halign">center</property>
```

Comment:
I'd make the output, at least, fill the window.

![Capture d’écran de 2020-12-11 14-03-07](https://user-images.githubusercontent.com/86760/101912429-a2211e00-3bb9-11eb-9691-d3d972ba12c3.png)

Reply:
The change on this branch makes the output fill the window horizontally, but not vertically (in my quick test).

I tried this patch:

```
diff --git a/src/window.ui b/src/window.ui
index 9469ad7..316b6e1 100644
--- a/src/window.ui
+++ b/src/window.ui
@@ -17,7 +17,7 @@
           <property name="orientation">vertical</property>
           <property name="spacing">5</property>
           <property name="margin">10</property>
-          <property name="valign">center</property>
+          <property name="valign">fill</property>
           <property name="halign">fill</property>
           <child>
             <object class="GtkLabel" id="chooser_label">
```

Here are the results. I think the initial appearance is worse, with stuff effectively top-justified:

![Capture d’écran de 2020-12-15 17-44-57](https://user-images.githubusercontent.com/86760/102252084-8beac900-3efd-11eb-88bd-dee4a44dc983.png)

But once you have some output you can resize the window and the output fills the window in both axes:

![Capture d’écran de 2020-12-15 17-44-51](https://user-images.githubusercontent.com/86760/102252087-8c835f80-3efd-11eb-9b14-a34561854541.png)

Reply:
But, it's a nitpick!